bonus.md

```diff
@@ -3,11 +3,16 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+Hash.fetch is useful to get value if key exists otherwise returning a default value.
 
 2. What is the disadvantage of single inheritance, and what's a possible solution to that problem?
+Classes can only inherit from a single other class although it may possibly be a part of multiple classes. Therefore the behaviors and attributes of the class will not be as DRY as it can be.
 
```

Comment:
Also, while you can only inherit from one class, that class can inherit from another class, and so on.